### ISSUE TITLE
GetMouseFocus Error on version 11

### ISSUE THREAD

**Issue by Fil1965** [TheGeneticsGuy/Guild-Roster-Manager---Group-Info#7]
Last version corrected this issue.

**Comment by TheGeneticsGuy** [comment]
Oh you know what, I just noticed this error in Group Info module - I was only looking at the core GRM!

Just a heads up, I noticed in my email you had linked a file you had updated as well. That's fine. If you ever want credit for something here on Github, feel free to do a pull request and I am more than happy to consider any code changes. GetMouseFoci() is a little different than GetMouseFocus() before. It's actually pretty nice new function I think, Also, it now places the same frame that GetMouseFocus() pulled as array position 1. No big deal.

Much easier to dig into the underlying frames. There's a problem though in a unified single codebase, since I release on Cata 4.4.0 Classic build as well, which is not yet unified with these changes. To do this I do a check like this:

```
GRM.GetMouseFocus = function ( frame )
    if GetMouseFoci then
        local c = GetMouseFoci();
        if c and c[1] then
            return c[1] == frame;
        end
    else
        return GetMouseFocus() == frame;
    end
end
```

So, while just replacing GetMouseFocus() with GetMouseFoci() technically works because Lua tables are what we call "truthy" as in, it returns true because it exists, it doesn't quite cover the full functionality. Thus, I use the above function to replace the old way and not only does it support the new function by Blizz, it also is compatible with Cata Classic until Blizz unifies that back end (and possibly 1.15.3 CE as well, as that might not yet have the update either, not sure).

With that being said, I really appreciate your efforts here and also reporting, so I just wanted to share with you the ultimate solution as I thought you might find it interesting.

Have a good one!
